.travis.yml

```
@@ -8,7 +8,7 @@ matrix:
         - php: 7.2
 
 before_install:
-    - export FLOW_TARGET_VERSION=6.0
+    - export FLOW_TARGET_VERSION=6.3
```

```suggestion
    - export FLOW_TARGET_VERSION=7.0
```
Otherwise it won't work, right?